static/app/utils/useOverlay.tsx

```diff
@@ -309,6 +319,15 @@ function useOverlay({
     overlayRef
   );
 
+  // Force popper update when elements mount/update
```

Comment:
We can probably expand on the comment here and explain that sometimes the popper can be misaligned on the initial render